src/Frontend/Parameters/PropertyParameterMap.cs

```diff
@@ -34,7 +34,8 @@ static PropertyParameterMap()
 				{ $"{reportSummary}.{nameof(ReportSummary.CpuBrand)}", "cb" },
 				{ $"{reportSummary}.{nameof(ReportSummary.CpuCores)}", "cc" },
 				{ $"{reportSummary}.{nameof(ReportSummary.CpuProcessors)}", "cp" },
-				{ $"{reportSummary}.{nameof(ReportSummary.OsPlatform)}", "op" },
+                { $"{reportSummary}.{nameof(ReportSummary.OsDistro)}", "od" },
```

Comment:
Is the indentation ok? It looks different from previous values

Reply:
Good catch, thanks. Should be fixed now.